SQL17thOct/FoodDeliveryDB

```diff
@@ -0,0 +1,90 @@
+CREATE DATABASE Food_Delivery_Service_Final;
+USE Food_Delivery_Service_Final;
+CREATE Table Customers(
+                        CustomerID int PRIMARY KEY,
+                        CustomerName varchar(30),
+					 	CustomerContact int,
```

Comment:
if this is phone number, please use varchar data type . since it won't contain more than 20 characters.

Reply:
ok, understood , now on will follow that while doing